README.md

```diff
@@ -214,6 +214,21 @@ podman container stop davincibox
 toolbox rm davincibox
 ```
 
+## Troubleshooting
+### No Audio output
+DaVinci Resolve uses ALSA. In `davincibox` this is supported via the `pipewire-alsa` plugin, which re-directs sound to pipewire.
+This however requires that the host system provides a pipewire server. If your host-system uses some other sound server, this might not work.
+
+You can either convert your host system to provide `pipewire` or change your `davincibox` instance to use any sound server you have running.
+
+For example, if your host-system uses `pulse-audio`, you can change `davincibox` as follows:
+
+```
+> distrobox-enter -n davincibox -- bash
```

Comment:
`distrobox-enter -n davincibox` on its own should invoke bash already, there shouldn't be a need to do so manually here

Reply:
done

Reply:
it actually gives me fish :-) nice :-) didn't think it is installed in the image

Reply:
That's strange. Fish isn't included, so it must be something else on your setup that's causing that.